src/main/java/net/sf/mzmine/desktop/impl/MainMenu.java

```diff
@@ -94,6 +94,7 @@
     add(projectMenu);
 
     lastProjectsSub = new JMenu("Open last project ...");
+    lastProjectsSub.setVisible(false);
```

Comment:
I think it's better to disable it using setEnabled(false).. disappearing menu items are confusing.